spec/import_spec.rb

```diff
@@ -44,6 +44,40 @@ def upload_file!(name, ext='csv')
     find_button('Import').click
   end
 
+  context "posts index" do
+    before do
+        Author.create!([{name: 'John', last_name: 'Doe'}, {name: 'Jane', last_name: 'Roe'}])
```

Comment:
Line is too long. [92/80]<br>Use 2 (not 4) spaces for indentation.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.